main.go

```diff
@@ -73,6 +73,7 @@ func setupFlags(cmd *cobra.Command) {
 	flagSet.BoolVar(&cfg.IsCriEnabled, "enable-cri", false, "Specify whether enable the cri part of pouchd which is used to support Kubernetes")
 	flagSet.StringVar(&cfg.CriConfig.CriVersion, "cri-version", "v1alpha2", "Specify the version of cri which is used to support Kubernetes")
 	flagSet.StringVar(&cfg.CriConfig.Listen, "listen-cri", "unix:///var/run/pouchcri.sock", "Specify listening address of CRI")
+	flagSet.BoolVar(&cfg.CriConfig.ServerTLS, "server-tls", false, "Specify whether enable https server of cri stream server")
```

Comment:
s/whether enable/whether to enable ?

Reply:
All above mentioned has been updated.  :)